### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by tomas-u** [dotnet/docs-maui#1915]
### Type of issue

Missing information

### Description

the link: .NET Maui Multiplattform App Sign-in user - Code samples
https://learn.microsoft.com/en-us/samples/azure-samples/ms-identity-docs-code-dotnet/ms-identity-docs-code-csharp-maui/?source=recommendations

is broken and gives a 404 - Page not found result

### Page URL

https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/communication/authentication?view=net-maui-8.0&tabs=android

### Content source URL

https://github.com/dotnet/docs-maui/blob/main/docs/platform-integration/communication/authentication.md

### Document Version Independent Id

829bc017-4a10-6ca5-0717-e1e62f59aacc

### Article author

davidbritch

### Metadata

* ID: 829bc017-4a10-6ca5-0717-e1e62f59aacc 
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**

---
[Associated WorkItem - 186536](https://dev.azure.com/msft-skilling/Content/_workitems/edit/186536)

**Comment by davidbritch** [comment]
Hi @tomas-u 

I'm a bit lost with this one. [This page](https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/communication/authentication?view=net-maui-8.0&tabs=android) doesn't link to https://learn.microsoft.com/en-us/samples/azure-samples/ms-identity-docs-code-dotnet/ms-identity-docs-code-csharp-maui/?source=recommendations

**Comment by davidbritch** [comment]
Now I see what you mean. The additional resources column on the right hand side links to the sample, which 404s:

![image](https://github.com/dotnet/docs-maui/assets/8092460/7bb745d7-a7af-4026-9bab-bc4a68a1b024)

This is problematic because that column isn't part of the doc source. It's generated by learn.microsoft.com. I'll try and track down where it's coming form.

**Comment by davidbritch** [comment]
So the code the sample link should be pointing to is https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/tree/main/xplat-maui

Alternatively, you might be better looking at: https://github.com/Azure-Samples/ms-identity-dotnetcore-maui

**Comment by davidbritch** [comment]
I've created a [PR](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/50) to update the sample description. Once merged this will cause the samples pipeline to re-ingest the sample and any links to it should be updated.

I've also created another [PR](https://github.com/Azure-Samples/ms-identity-dotnetcore-maui/pull/13) to cause the ADB2C sample to show under the .NET MAUI filter on the samples browser.
